velox/connectors/hive/HiveDataSource.cpp

```diff
@@ -490,6 +493,7 @@ HiveDataSource::HiveDataSource(
   }
 
   readerOpts_.setFileSchema(hiveTableHandle->dataColumns());
+  readerOpts_.setFileColumnNamesReadAsLowerCase(fileColumnNamesReadAsLowerCase);
```

Comment:
We can simplify the constructor of `HiveDataSource` here, by setting this in `HiveConnector::createDataSource` and remove the `fileColumnNamesReadAsLowerCase` parameter.

Reply:
Makes sense. Updated. Could you review again? Thank you.